libc-top-half/musl/src/wasix/context.c

```diff
@@ -0,0 +1,42 @@
+#include <errno.h>
+#include <wasi/api_wasi.h>
+#include <wasi/api_wasix.h>
+#include <wasix/context.h>
+
+wasix_context_id_t __wasix_context_main(void) {
+  return 0; // Main context is always ID 0 for now
```

Comment:
Main context = zero can be part of the spec

Reply:
We might want to change that at a later point if we need better support for forking (e.g. main context will have the context id that the fork happened on). I wanted to keep the possibility open to have other values than 0.